src/kubernetes-ingress-controller/deployment/kong-enterprise.md

```diff
@@ -87,8 +87,7 @@ postgres                  ClusterIP      10.63.241.104   <none>          5432/TC
 > Note: Depending on the Kubernetes distribution you are using, you might or might
 not see an external IP assigned to the three LoadBalancer type services. Please see
 your provider's guide on obtaining an IP address for a Kubernetes Service of
-type `LoadBalancer`. If you are running Minikube, you will not get an
-external IP address.
+type `LoadBalancer`.
```

Comment:
There are some cases where you will not get an external IP address using Minikube.

Can we add more information about how to get an external IP address when using Minikube?

Reply:
External IP is always going to be None, unless you deploy the Service as type: LoadBalancer. If you do that, you have to run minikube tunnel to get access to the service.
Ref: https://github.com/kubernetes/minikube/issues/3966

WDYT?
// @mheap

Reply:
The page seems accurate as-is. If we want to add more minikube-specific instructions, feel free to raise a PR with those changes